packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-testing.md

```diff
@@ -3,83 +3,79 @@ id: Testing your accessibility
+    - Text color against background color ([Understanding WCAG 1.4.3](//www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html))
+    - Text color against link color ([Technique G183](//www.w3.org/TR/WCAG20-TECHS/G183.html))
+    - Visible boundaries of buttons and form elements against adjacent background color ([Understanding WCAG 1.4.11](//www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html))
 
+## Accessibility testing checklist
 
-<br />
+We recommend referencing the following checklist as you assess your product's accessibility.
 
-### Checklist for testing
-These are some of the main areas we check for on the PatternFly team. We recommend that you check for these same areas in your applications, too. 
+These are some of the main areas we check for on the PatternFly team to ensure that components meet consistent accessibility standards, but, to check your complete implementation, we recommend checking these same areas in your products. 
 
-**General questions**
+### General criteria
 
 <label><input type="checkbox" /> VO Rotor Navigation can discover all information. </label><br/>
 <label><input type="checkbox" /> VO Shortcut Navigation can discover all information. </label><br/>
-<label><input type="checkbox" /> VO Cursor Navigation can discover all information (dependent on component and use case- easy way to navigate to component). </label><br/>
+<label><input type="checkbox" /> VO Cursor Navigation can discover all information (dependent on component and use case — there should be an easy way to navigate to the component). </label><br/>
 <label><input type="checkbox" /> Tabbed Keyboard Navigation can discover all information. </label>
-- However, if content should be hidden from a screen reader, for instance, if it’s offscreen or just presentational, make sure that content is set to aria-hidden=”true”.  <br/>
+- Unless content should be hidden from a screen reader (such as offscreen or presentational information) make sure that content is set to aria-hidden=”true”.  <br/>
 
-<label><input type="checkbox" /> Is it understandable and usable? </label><br/>
-  - <label><input type="checkbox" /> When you navigate to an item by keyboard or by screen reader, you can easily understand and use the item. </label><br/>
+<label><input type="checkbox" /> Items are understandable and usable. </label><br/>
+  - <label><input type="checkbox" /> When you navigate to an item by keyboard or by screen reader, you should be able to easily understand and use the item. </label><br/>
   - <label><input type="checkbox" /> The flow of information makes sense when navigating. Screen readers navigate the page in DOM order. If you used CSS to visually reposition elements, they might be announced in a nonsensical sequence. Instead, if you need something to appear earlier in the page, try to physically move it earlier in the DOM.</label>
```

Comment:
So I only have an example of a change for this section of checklist items, but the same updates could be made to the next section as well:

```suggestion
<Checkbox label="Rotor navigation can discover all information, if a rotor menu exists." id="general-criteria-1" />
<Checkbox label="Shortcut navigation can discover all applicable information." description="For example, a keyboard shortcut that navigates between all headings on a page should discover all intended headings." id="general-criteria-2" />
<Checkbox label="Navigation via the assistive technology can discover all applicable information." description="Some assistive technologies will have their own means of navigation and focus management. For example, VoiceOver uses a 'VO' key plus right or left arrow keys to navigate a page. This may be different than conventional keyboard navigation." id="general-criteria-3" />
<Checkbox label={<span>Keyboard navigation via the <kbd>Tab</kbd> key can discover all information.</span>} description={<span>If content should be hidden from other assistive technology like a screen reader, <span className="ws-code">aria-hidden="true"</span> should be passed to it.</span>} id="general-criteria-4" />
<Checkbox label="Items are understandable and usable." body={<><Checkbox label="When you navigate to an item by keyboard or other assistive technology, you should be able to easily understand and use the item." id="general-criteria-5a" /><Checkbox label="The flow of information makes sense when navigating." description="Assistive technology like screen readers navigate the page in DOM order. If you used CSS to visually reposition elements, they might be announced in a nonsensical sequence. Instead, if you need something to appear earlier in the page, try to physically move it earlier in the DOM.
" id="general-criteria-5b" /></>} id="general-criteria-5" />
```

Some verbiage tweaks, but also using the PF checkbox component instead of the native HTML one. It requires a little more setup to use, especially when it comes to using other HTML tags and links, but it results in the following:

![General criteria checklist with PatternFly Checkbox](https://github.com/patternfly/patternfly-org/assets/70952936/f4539f19-fecb-4430-8990-d4ea19636a45)

Reply:
for your suggestions on line 73 & 74 - it looks like some characters aren't allowed within <Checkbox>, like code snippets & "body={<>" - at least for me. Did you create your screenshot via vs code, without any errors?